internal/engine/evaluator.go

```diff
@@ -391,9 +391,11 @@ func toSet(values []string) stringSet {
 }
 
 func setIntersects(s1 protoSet, s2 stringSet) bool {
```

Comment:
Let's rename and move this function to the file with the glob functions in the `util` package so  that it is obvious that it uses glob matching.